tools/roslaunch/src/roslaunch/__init__.py

```diff
@@ -363,6 +368,8 @@ def main(argv=sys.argv):
             try: os.unlink(options.pid_fn)
             except os.error: pass
 
+    print('exiting from main() {}'.format(exit_code))
```

Comment:
This looks like a debug print that can be removed - people who are interested in the exit code of the executable can get this from the shell.